### PR TITLE
Create do-you-speak-prestashop-january-2023.md

### PR DESCRIPTION
**Pull request by l-delin**

### CODE REVIEW

**Comment by l-delin** [comment]
ping @coffrep

File: content/news/translation/2023/do-you-speak-prestashop-january-2023.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+title: "Do you speak PrestaShop? – January 2023 edition"
+subtitle: "Where you'll learn about a new language"
+date: 2023-03-02
+authors: [LéaDelin]
+icon: icon-world
+image: /assets/images/2019/01/Build-Crowdin-banner.jpg
+tags: [translation, i18n, l10n]
+---
+
+Contributing to PrestaShop is not only about the code, it is also about taking part in the [PrestaShop translation project](https://crowdin.com/project/prestashop-official)! This report will tell you how the software translations evolved in December!
```
Comment (coffrep) [inline content/news/translation/2023/do-you-speak-prestashop-january-2023.md]:
```suggestion
Contributing to PrestaShop is not only about the code, it is also about taking part in the [PrestaShop translation project](https://crowdin.com/project/prestashop-official)! This report will tell you how the software translations evolved in January!
```

File: content/news/translation/2023/do-you-speak-prestashop-january-2023.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+Thanks to [Ali Zhumagaliev](https://crowdin.com/profile/alizhumagaliev), the Karakalpak language is now available for translation on the Crowdin project!
+
+### 💕 Many thanks to January’s newcomers
+
+Many people subscribe to the project(s) of their choice each month, but not so many start translating straight away. 
+For that reason, I would like to thank [OnlyHardOfficial](https://crowdin.com/profile/onlyhardofficial), [Akarawuth Tamrareang (@JoomlaCorner)](https://crowdin.com/profile/joomlacorner) and [Ali Zhumagaliev](https://crowdin.com/profile/alizhumagaliev), who rushed into the efforts immediately.👏
+
+### 📖 The translation guide is there to help
+
+Ever wondered how the translation process works? or how to become a proofreader? The translation guide is there to help. You can find it [here](https://docs.prestashop-project.org/translating-prestashop/).
```
Comment (coffrep) [inline content/news/translation/2023/do-you-speak-prestashop-january-2023.md]:
```suggestion
Ever wondered how the translation process works? Or how to become a proofreader? The translation guide is there to help. You can find it [here](https://docs.prestashop-project.org/translating-prestashop/).
```

File: content/news/translation/2023/do-you-speak-prestashop-january-2023.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+
+Congratulations, and welcome to the project's new top contributors! :clap:
+ 
+Remember, you can see who has been contributing to our translation project thanks to the [Translators page](https://translators.prestashop.com/).
+ 
+## Complete translations
+ 
+#### Fully available languages
+ 
+At the end of December, PrestaShop 8 was fully available (= 100% translated and validated) in **14 languages**. That’s great!
```
Comment (coffrep) [inline content/news/translation/2023/do-you-speak-prestashop-january-2023.md]:
```suggestion
At the end of January, PrestaShop 8 was fully available (= 100% translated and validated) in **14 languages**. That’s great!
```

File: content/news/translation/2023/do-you-speak-prestashop-january-2023.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+
+| Catalan | Chinese (CN) | Chinese (TW) | Czech |
+| Dutch | French | Greek | Hungarian | Italian | 
+| Portuguese (BR) | Spanish (ES) | Ukrainian | 
+
+ Thank you for your involvement. :tada:
```
Comment (coffrep) [inline content/news/translation/2023/do-you-speak-prestashop-january-2023.md:79]:
```suggestion
 Thank you for your involvement. :tada:
 
```

File: content/news/translation/2023/do-you-speak-prestashop-january-2023.md [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+#### Almost there!
+
+Korean, Lithuanian, Romanian, and Polish are almost fully available. There is only 1% of strings to translate and proofread! 💪
+
+## Languages with the best evolution
+
+The following languages had the best progress in December thanks to the translation community:
```
Comment (coffrep) [inline content/news/translation/2023/do-you-speak-prestashop-january-2023.md]:
```suggestion
The following languages had the best progress in January thanks to the translation community:
```

**Review by coffrep — CHANGES_REQUESTED** [review]
Quelques suggestions - j'ai juste suggéré de modifier le mois à quelques reprises, je ne sais pas si ca a du sens pour toi ?

**Review by coffrep — APPROVED** [review]
Thanks for the great post! As mentioned on Slack, super if this post is published beginning of next week - thanks Léa.

File: content/news/translation/2023/do-you-speak-prestashop-january-2023.md [hunk 6]
```diff
@@ -0,0 +1,109 @@
+---
+layout: post
+aliases: ["/news/do-you-speak-prestashop-january-2023"]
+title: "Do you speak PrestaShop? – January 2023 edition"
+subtitle: "Where you'll learn about a new language"
+date: 2023-03-02
```
Comment (kpodemski) [inline content/news/translation/2023/do-you-speak-prestashop-january-2023.md]:
```suggestion
date: 2023-02-13
```

:)

Comment (coffrep) [reply content/news/translation/2023/do-you-speak-prestashop-january-2023.md]: Perfect 👍

**Comment by l-delin** [comment]
Hello, could we get this published? Thank you :-)

**Review by kpodemski — APPROVED** [review]

**Comment by kpodemski** [comment]
Thanks @l-delin

**Pull request merged**